examples/subscriptions/storefrontapi.generated.d.ts

```diff
@@ -261,6 +261,24 @@ export type ProductFragment = Pick<
     >;
   };
   seo: Pick<StorefrontAPI.Seo, 'description' | 'title'>;
+  sellingPlanGroups: {
```

Comment:
Do we need to put the generated types file in the example commits? Would codegen take care of this?

Or perhaps we leave it as-is in the commit, but I don't think it's a file we need to address specifically in the Readme 🤔